mava/wrappers/open_spiel.py

```diff
@@ -17,11 +17,15 @@
 
 import dm_env
 import numpy as np
-import pyspiel  # type: ignore
+
+try:
+    import pyspiel  # type: ignore
+    from open_spiel.python import rl_environment  # type: ignore
+except ModuleNotFoundError:
```

Comment:
Nice, not sure if this is the reason, but we should perhaps consider this for other environments as well for when they haven't been installed as dependencies.

Reply:
I did a quick pass and did this for the other environments.